webapp/components/DeleteData/DeleteData.vue

```diff
@@ -9,23 +9,35 @@
     </label>
     <ds-input v-model="enableDeletionValue" />
     <p class="notice">{{ $t('settings.deleteUserAccount.accountDescription') }}</p>
```

Comment:
> [<img alt="mattwr18" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/mattwr18) **Authored by [mattwr18](https://github.com/mattwr18)**
_<time datetime="2020-04-30T13:25:54Z" title="Thursday, April 30th 2020, 3:25:54 pm +02:00">Apr 30, 2020</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -1,5 +1,5 @@
 <template>
-  <base-card class="delete-data">
+  <base-card v-if="currentUser.role !== 'admin'" class="delete-data">
```


I don't think we have agreed on this, therefore I would open an issue and we can discuss it, not just make this decision alone.

Reply:
> [<img alt="ogerly" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/ogerly) **Authored by [ogerly](https://github.com/ogerly)**
_<time datetime="2020-04-30T14:02:49Z" title="Thursday, April 30th 2020, 4:02:49 pm +02:00">Apr 30, 2020</time>_

---


so i changed it